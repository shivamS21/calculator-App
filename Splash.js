
import 'react-native-gesture-handler';
import React, { Component,useState } from 'react';
import { Animated ,View, Text,TextInput,StyleSheet,Image, ImageBackground } from 'react-native';
import { NavigationContainer } from '@react-navigation/native';
import { createStackNavigator } from '@react-navigation/stack';
import { createAppContainer,NavigationActions } from 'react-navigation';
export default class Splash extends Component {
    state = {
        text: "",
    }
    handletext = (content) => {
        this.setState({ text: content })
    }
    render(){
        return (
            <View style={{flex:1,backgroundColor:'#e6ffff',justifyContent:'center',alignItems:'center'}}>
                <ImageBackground source={require('./images/ff.jpg')} resizeMode = 'cover' style={{width:'100%', height:'100%',flex:1}}>
                    <TextInput style = {styles.input} 
                    underlineColorAndroid = "transparent" 
                    placeholder = "write your views here" 
                    placeholderTextColor = "white" 
                    autoCapitalize = "none" 
                    onChangeText = {this.handletext}
                    onSubmitEditing={()=>this.props.navigation.navigate('story',{paramKey:this.state.text})}/>
                </ImageBackground>
            </View>
          );
    }
}
const styles = StyleSheet.create({
    input: {
        textAlign:'center',
        height: 43,
        width:325,
        borderColor: '#ccf2ff',
        borderWidth: 2,
        bottom:0,
        top:330,
        fontSize:20,
        fontStyle:'italic',
        left:40,
        // backgroundColor:'#d9d9f2',
        color:'white',
        opacity:0.7,
        borderRadius:30
    },
})