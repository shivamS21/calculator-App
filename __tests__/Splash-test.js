import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Splash from '../Splash';

it('checking test once again',()=>{
    const home = renderer.create(<Splash/>).toJSON();
    expect(home).toMatchSnapshot();
})