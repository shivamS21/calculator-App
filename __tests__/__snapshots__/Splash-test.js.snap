// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checking test once again 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#e6ffff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "flex": 1,
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <Image
      resizeMode="cover"
      source={
        Object {
          "testUri": "../../../images/ff.jpg",
        }
      }
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": "100%",
            "width": "100%",
          },
          undefined,
        ]
      }
    />
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="write your views here"
      placeholderTextColor="white"
      rejectResponderTermination={true}
      style={
        Object {
          "borderColor": "#ccf2ff",
          "borderRadius": 30,
          "borderWidth": 2,
          "bottom": 0,
          "color": "white",
          "fontSize": 20,
          "fontStyle": "italic",
          "height": 43,
          "left": 40,
          "opacity": 0.7,
          "textAlign": "center",
          "top": 330,
          "width": 325,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
</View>
`;
